specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/cognitiveservices.json

```diff
@@ -870,6 +874,17 @@
           "$ref": "#/definitions/NetworkRuleSet",
           "description": "A collection of rules governing the accessibility from specific network locations."
         },
+        "encryption": {
```

Comment:
This constitutes a breaking change and requires a new API version.

See 'New property added to response'
https://microsoft.sharepoint.com/:w:/r/teams/azureresourcemanagerteam/_layouts/15/guestaccess.aspx?e=xEcppo&share=EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw

Reply:
I randomly opened a few recent PRs from other teams, they all adding properties to a response without creating a new version.
https://github.com/Azure/azure-rest-api-specs/pull/8498/files
https://github.com/Azure/azure-rest-api-specs/pull/8508/files
https://github.com/Azure/azure-rest-api-specs/pull/8468/files

The GET-PUT story doesn't apply in here, as our SDK update operation will ignore the missing properties and consider it as "no change".